lib/index.js

```diff
@@ -200,84 +180,56 @@ const addToNameCal = (food) => {
     </div>`
 }
 
-const prependToBreakfast = (food) => {
-  $('.breakfasts').prepend(`
-    <article class='breakfast' data-breakfast-food-id=${food.id}>` + addToNameCal(food) +
-      `<p class='delete'><button data-breakfast-food-id=${food.id} class='delete-food-btn'><i class="fa fa-trash"></i></button></p>
-    </article>`
-  )
-}
-
-const prependToLunch = (food) => {
-  $('.lunches').prepend(`
-    <article class='lunch' data-lunch-food-id=${food.id}>` + addToNameCal(food) +
-      `<p class='delete'><button data-lunch-food-id=${food.id} class='delete-food-btn'><i class="fa fa-trash"></i></button></p>
-    </article>`
-  )
-}
-
-const prependToDinner = (food) => {
-    $('.dinners').prepend(`
-      <article class='dinner' data-dinner-food-id=${food.id}>` + addToNameCal(food) +
-        `<p class='delete'><button data-dinner-food-id=${food.id} class='delete-food-btn'><i class="fa fa-trash"></i></button></p>
-      </article>`
-    )
-}
-
-const prependToSnack = (food) => {
-  $('.snacks').prepend(`
-    <article class='snack' data-snack-food-id=${food.id}>` + addToNameCal(food) +
-      `<p class='delete'><button data-snack-food-id=${food.id} class='delete-food-btn'><i class="fa fa-trash"></i></button></p>
+const prependToMeal = (food, meal) => {
+  var mealType = meal.name.toLowerCase()
```

Comment:
Since you are using ES5 syntax elsewhere, I would use ES6 syntax for declaring variables `const` & `let`

Reply:
ok! so I would literally just change `var` to `let`?

Reply:
Yup! So here you would `const` since this variable will not change but below I would declare pluralMeal before the if statement and use `let` as it can change:
```
let pluralMeal;

if (mealType === "lunch") {
  pluralMeal = mealType + 'es'
...
```

Reply:
Awesome! Just implemented it. Thanks for the heads up!